<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Redactor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyImages.qrc">
    <normaloff>:/Icons/Images_lab4/icon1.ico</normaloff>:/Icons/Images_lab4/icon1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;SpinBuddy</string>
        </property>
        <property name="buddy">
         <cstring>spinBox</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="toolTip">
         <string comment="Value %"/>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;SliderBuddy</string>
        </property>
        <property name="buddy">
         <cstring>horizontalSlider</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Exit</string>
      </property>
      <property name="icon">
       <iconset resource="MyImages.qrc">
        <normaloff>:/Pictures/Images_lab4/Disney_42.png</normaloff>:/Pictures/Images_lab4/Disney_42.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;EditBuddy</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="MyPicture" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Information">
    <property name="title">
     <string>&amp;Information</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Information"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string>Нечего не делаю...</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Exit"/>
  </widget>
  <action name="action_Exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MyImages.qrc">
     <normaloff>:/Pictures/Images_lab4/spyder1.bmp</normaloff>:/Pictures/Images_lab4/spyder1.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPicture</class>
   <extends>QWidget</extends>
   <header>mypicture.h</header>
   <container>1</container>
   <slots>
    <signal>wheelsignal(int)</signal>
    <slot>scaleChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyImages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>myslot(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>mysignal(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>scaleChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>action_Exit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>wheelsignal(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>mysignal(int)</signal>
  <slot>myslot(QString)</slot>
 </slots>
</ui>
